config/cache.php

```diff
@@ -12,7 +12,9 @@
 	| using this caching library. This connection is used when another is
 	| not explicitly specified when executing a given caching function.
 	|
-	| Supported: "apc", "array", "database", "file", "memcached", "redis"
+
+	| Supported: "apc", "array", "database", "file",
+	|			"memcached", "redis", "dynamodb"
```

Comment:
Not a big issue, but this line wrapping seems weird.

Reply:
that is actually the file from the lastest Laravel release. :)